src/components/BookPlace.tsx

```diff
@@ -133,23 +167,37 @@ const BookPlace: React.FC<BookPlaceProps> = ({ onClose, placeId, placeBookings,
           label='Forever booking'
         />
         <Button disabled={disabledFlag} onClick={onBookingDatesOpen}>
-          {bookingDateRange ?
-            `Booked Days: ${dateToString(bookingDateRange.startDate)} - 
-            ${bookingDateRange?.endDate ? dateToString(bookingDateRange.endDate) : 'Forever'}` :
-            'Choose Booking Dates'
+          {bookingDateRange
+            ? `Booked Days: ${dateToString(bookingDateRange.startDate)} - 
+            ${bookingDateRange?.endDate ? dateToString(bookingDateRange.endDate) : 'Forever'}`
+            : 'Choose Booking Dates'
           }
         </Button>
-        <InputGroup 
+        <select
+          value={visitorName}
           required
+          onChange={onSelectUserChange}
           disabled={disabledFlag}
-          placeholder='Name'
-          value={visitorName}
-          onChange={onNameChange}
-        />
+        >
+          <option value={placeholderValue} selected>Choose user</option>
+          {members.map(member =>
+            <option key={member.id} id={member.id} value={JSON.stringify({ name: member.data().name, id: member.id })}>
+              {member.data().name}
+            </option>
+          )}
+          <option value={addMemberValue}>{addMemberValue}</option>
+        </select>
+        <Dialog
+          isOpen={isAddMemberOpen}
+          onClose={onAddMemberClose}
```

Comment:
While `onClose` is a good property name, as it describes which event is triggered by a component, it is not good as a function name because it doesn't describe what it does.